4-Express/src/index.js

```diff
@@ -0,0 +1,220 @@
+    list.appendChild(itemElement)
+
+    clearMenu()
+    setInnerText(constants.applyButtonID,'Add')
+}
+function deleteCheckedItems(){
+    const itemList = storage.getAll()
+    const list = document.getElementById(constants.todoListID)
+    for (const id in itemList) {
+        const checkbox = document.getElementById(`itemCheckbox${id}`)
+        if(checkbox.checked){
+            const itemElement = document.getElementById(`${id}`)
+            list.removeChild(itemElement)
+            storage.remove(id)
+        }
+    }
+}
+function cancelEdit(){
+    clearMenu()
+    editMode = false
+    setInnerText(constants.applyButtonID,'Add')
+    document.getElementById(constants.cancelButtonID).hidden = true
+    editedID = 0  
+}
+
+function clearMenu(){
+    setValue(constants.titleInputID,'')
+    setValue(constants.contentInputID,'')
+}
+
+function menuButtonClick(){
+    if(!editMode){
+        addItem()
+    }else{
+        if (getValue(constants.titleInputID)==''){
+            alert('cant set an empty title')
+            return
+        }
+        const title = getValue(constants.titleInputID)
+        const content = getValue(constants.contentInputID)
+        setInnerText(`title${editedID}`,title)
+        setInnerText(`content${editedID}`,content)
+        storage.set(editedID, title, content)
+        clearMenu()
+        editMode = false
+        setInnerText(constants.applyButtonID,'Add')
+        document.getElementById(constants.cancelButtonID).hidden = true
+        editedID = 0
+    }
+}
+function addButton(){
+    clearMenu()
+    document.getElementById(constants.titleInputID).focus()
+}
+
+function createTodoItemElement(itemID,title,content){
+
+    const itemElement = document.createElement('ol')
+    itemElement.classList.add(classes.todoItem)
+    itemElement.id = itemID;
+
+    const itemCheckboxElement = createItemCheckboxElement(itemID)
+    const itemTextElement = createItemTextElement(itemID,title,content)
+    const itemButtonsElement = createItemButtonsElement(itemID)
+
+    itemElement.appendChild(itemCheckboxElement)
+    itemElement.appendChild(itemTextElement)
+    itemElement.appendChild(itemButtonsElement)
+
+    return itemElement
+}
+function createItemCheckboxElement(itemID){
+    const itemCheckboxElement = document.createElement('input')
+    itemCheckboxElement.type = 'checkbox'
+    itemCheckboxElement.classList.add(classes.itemCheckbox)
+    itemCheckboxElement.id = `itemCheckbox${itemID}`
+    itemCheckboxElement.onclick = function(){
+        const titleElement = document.getElementById(`title${itemID}`)
+        const contentElement = document.getElementById(`content${itemID}`)
+        const editButtonElement = document.getElementById(`itemEdit${itemID}`)
+        if(titleElement.classList.contains(classes.strike)
+            || contentElement.classList.contains(classes.invisible)
+            || editButtonElement.classList.contains(classes.invisible)){
```

Comment:
not sure when this change was introduced, but generally it's a bad practice to mix classes with logic, as they are intended for styling purposes. use data-attributes/manage the application state with javascript